src/utils/helpers.js

```diff
@@ -165,7 +165,7 @@ export function createAbsoluteElement(el) {
 }
 
 export function isVueComponent(c) {
-    return c && c._isVue
+    return c && c.$ != null && c.$.vnode != null
 }
 
 /**
```

Comment:
Please remove the extra `*` here 😅

Reply:
@wesdevpro They are comments intended to be processed with a documentation generator such as [JSDoc](https://jsdoc.app). I believe people who are accustomed to that style ([javadoc](https://en.wikipedia.org/wiki/Javadoc) style), like me, tend to write comments in that form without clear plan to run a documentation generator.